src/main/pages/user-management/permissions.adoc

```diff
@@ -11,11 +11,9 @@ folder: user-management
 [id="overview"]
 == Overview
 
-Permissions are used to control user actions. Rather than providing a fixed set of roles we use a broader set of permissions that can be applied in any combination of objects to establish the security model you need.
+Permissions are used to control the actions of users and establish a security model. You can define "who" is allowed to do "what". You control resources managed by Che and allow certain actions by assigning permissions to users.  
 
-Che also provides a mechanisms and layers which allow to define "who" is allowed to do "what". Any user and administrator can control resources managed by Che and allow certain actions or behaviors for other users or groups. For example as workspace owner you can grant other users permission to see and/or use your workspace.
-
-Permissions can be applied to:
+Permissions are applied to the following:
```

Comment:
I thought that since it is a list that an introduction is needed with  "to the following".  I thought "may" and "can" should be avoided but I see that can is ok...  How about:  Permissions can be applied to the following:

Reply:
I have no problem with "to the following", though I think it's redundant. But you changed "can be applied" to "are applied". So, the meaning shifted from "it's possible to do something" to "something happens".